online-shop/src/app/auth.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable } from '@angular/core';
+import {Observable, of} from "rxjs";
+import {HttpClient} from "@angular/common/http";
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class AuthService {
+
+  isLoggedIn = false;
+  readonly ROOT_URL = "http://localhost:3000"
```

Comment:
**suggestion:** you cand keep this url inside the `environment.ts` file or you could use a [proxy](https://angular.io/guide/build#proxying-to-a-backend-server)